projects/storefrontstyles/scss/themes/calydon/components/cart/_item-counter.scss

```diff
@@ -0,0 +1,5 @@
+%cx-item-counter-calydon {
```

Comment:
The item counter implementation has been merged (coming from 2.0 branch) and have taken a different journey for a11y. The code below is no longer used, we might need to re-evaluate.

Reply:
A ticket for this will be created.

Reply:
@tobi-or-not-tobi @kacperknapik Agreed and ticket #6713 created, it will be addressed during the `calydon-as-base` effort to avoid delaying the merge of this PR any further.